antarest/matrixstore/service.py

```diff
@@ -375,7 +427,11 @@ def create_matrix_files(
                 mtx = self.get(mid)
                 if not mtx:
                     continue
-                write_tsv_matrix(mtx, tmpdir)
+                name = f"matrix-{mtx.id}.txt"
+                filepath = f"{tmpdir}/{name}"
+                array = np.array(mtx.data, dtype=np.float64)
+                # noinspection PyTypeChecker
+                np.savetxt(filepath, array, delimiter="\t", fmt="%.18f")
```

Comment:
We could have a small utility method so that we don't have to repeat the same arguments every time we write a matrix (and know them :) )

Reply:
Sorry, no. I prefer to avoid one-line utility methods, because they're more readable in general.